### PR TITLE
[Feature](datatype) update be ut codes and fix bugs for IPv4/v6

### PR DESCRIPTION
**Pull request by sjyango**
## Proposed changes

update `from_string_test` be ut based on issue #26534

- update be ut codes
- fix bugs
- remove unused codes in DataTypeIPv4/v6

output:
image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sjyango** [comment]
run buildall

**Comment by sjyango** [comment]
run p0

**Comment by sjyango** [comment]
run p0

File: be/test/vec/data_types/from_string_test.cpp [hunk 1]
```diff
@@ -283,6 +283,57 @@ TEST(FromStringTest, ScalaWrapperFieldVsDataType) {
         }
     }
 
+    // ipv4 and ipv6 type
+    {
+        typedef std::pair<FieldType, string> FieldType_RandStr;
+        std::vector<FieldType_RandStr> ip_scala_field_types = {
+                FieldType_RandStr(FieldType::OLAP_FIELD_TYPE_IPV4, "127.0.0.1"),
+                FieldType_RandStr(FieldType::OLAP_FIELD_TYPE_IPV6, "2405:9800:9800:66::2")};
```
Comment (amorynan) [inline be/test/vec/data_types/from_string_test.cpp:291]: add max min case here. to make beut rung

File: be/src/vec/data_types/data_type_ipv6.cpp [hunk 2]
```diff
@@ -37,8 +37,9 @@ std::string DataTypeIPv6::to_string(const IColumn& column, size_t row_num) const
     auto result = check_column_const_set_readability(column, row_num);
     ColumnPtr ptr = result.first;
     row_num = result.second;
-    IPv6 value = assert_cast<const ColumnIPv6&>(*ptr).get_element(row_num);
-    return convert_ipv6_to_string(value);
```
Comment (amorynan) [inline be/src/vec/data_types/data_type_ipv6.cpp:41]: why write convert before ?

**Comment by sjyango** [comment]
run buildall

**Review by amorynan — APPROVED** [review]
LGTM

**Review by starocean999 — APPROVED** [review]

**Pull request merged**